.github/workflows/basic_reg_test.sh

```diff
@@ -1,115 +1,116 @@
 #!/bin/bash
 
 set -e
-
+source openfpga.sh
```

Comment:
Bias to the use of bash is risky to me. I prefer to see the use of python directly. Therefore, this script is portable. Currently, I am not sure if this script can be used in csh etc.

in future, this script should be replaced by a python script which run these tests in batch.

Reply:
Yeah its only for bash-like shells, in `csh` shell you don't need  this script. You can use the previous method you used to run tasks. this is just an enhancement incase you are using bash shell, which is true in the docker image we are releasing.